rust/azure_iot_operations_services/src/schema_registry/schemaregistry_gen/schema_registry/format.rs

```diff
@@ -3,7 +3,7 @@
 use serde::{Deserialize, Serialize};
 
 /// Supported schema formats
-#[derive(Serialize, Deserialize, Debug, Clone)]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
```

Comment:
can't change this in the generated files otherwise they'll get overwritten anytime we regenerate

Reply:
oh damn I just realized this was generated code....